packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -66,11 +66,11 @@ export class VaButton {
   @Prop({ reflect: true }) secondary?: boolean = false;
 
   /**
-   * A value of: `prevent` --will  triger the onsubmit callback on form, but not submit;
-   * `skip` --will submit but not trigger onsubmit callback; and `submit` or truthy -- will do both
-   * `false` acts a boolean false
+   *  Having this attribut present will set the type of this button as 'submit'.
+   *  A value of: `prevent` --will  triger the onsubmit callback on form, but won't submit the form;
```

Comment:
Missing a 'g' in 'trigger'. Also grammatically should probably be "will trigger the onsubmit callback on *the* form".